ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -277,7 +281,7 @@ runDataDiffusion tracers
         -- Return an IPv4 address with an emphemeral portnumber if we use IPv4
         anyIPv4Addr :: SockAddr -> Maybe SockAddr
         anyIPv4Addr (Socket.SockAddrInet _ _) = Just (Socket.SockAddrInet 0 0)
-        anyIPv4Addr _ = Nothing
+        anyIPv4Addr _                         = Nothing
```

Comment:
Please don't mix stylistic changes in the same commit, this will only make rebasing other branches harder.